### PR TITLE
Modules and swag

### PR DESCRIPTION
**Pull request by nmclrn**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,14 @@
-FROM golang:1.10
-MAINTAINER Greg Taschuk
-WORKDIR /go/src/github.com/TruSet/RevealerAPI
+FROM golang:1.13
+MAINTAINER Neil McLaren & Greg Taschuk
+WORKDIR /truset-revealer
```
Comment (gtaschuk) [inline Dockerfile]: if we're changing the working directory we need to also change the volumes being shared in the TruSet/TruSet docker-compose.yml to maintain the code sharing between host and container

File: main.go [hunk 2]
```diff
@@ -11,15 +11,41 @@ import (
...
+//
+// @produces json
+// @title TruSet Revealer API
+// @version 0.1
+// @description A REST interface for submitting votes (e.g. Accept or Reject) on proposed data records.
+// @termsOfService ToS TBD
+//
+// @contact.name API Support
+// @contact.url https://www.truset.com/contact/
+// @contact.email greg@truset.com
```
Comment (gtaschuk) [inline main.go]: can we make this my consensys email - or leave it out?   Who do you envision using this documentation?

File: .circleci/config.yml [hunk 3]
```diff
@@ -23,25 +23,23 @@ commands:
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.9
+      - image: circleci/golang:1.13
         environment:
           DEP_VERSION: 0.5.0
```
Comment (gtaschuk) [inline .circleci/config.yml]: I think we can remove this environment variable now

File: Dockerfile (hunk 1 above)
Comment (nmclrn) [reply Dockerfile]: https://github.com/TruSet/TruSet/pull/675

File: main.go (hunk 2 above)
Comment (nmclrn) [reply main.go]: Changed to ConsenSys.

> Who do you envision using this documentation?

Probably no one, but might one day be useful to someone who stumbles across consumer code and wants to discuss bugs/changes/feature requests.

**Pull request merged**